docs/design/storagenode-graceful-exit/pieces.md

```diff
@@ -0,0 +1,45 @@
+# Storage Node Graceful Exit - Gathering Pieces
+
+[Graceful Exit Overview](overview.md)
+
+## Abstract
+
+During Graceful Exit, a satellite needs to find pieces to be transferred from the exiting storage node.
+
+## Background
+
+Graceful Exit contains a process that moves existing pieces from one storage node to another. To accomplish this we need a list of pieces that need to be transferred.
+
+Pieces with lower durability have higher importance to be transferred.
+
+## Design
+
+We need a service on the satellite that finds pieces in the metainfo database that need to be transferred. We'll call this service `gexit.Service` or Graceful Exit service.
+
+The service starts by asking overlay for all exiting nodes where `nodes.exit_loop_completed` is false.
```

Comment:
`nodes.exit_loop_completed` is of type `timestamp`. So the check should be if it is `NULL`, not `false`.

Reply:
Updated